src/options.mjs

```diff
@@ -1,11 +1,66 @@
+import fs from "fs";
+import path from "path";
+
 const CATEGORY_PHP = "PHP";
 
+/**
+ * Detect minimum PHP version from the composer.json file
+ * @param def {string} Default PHP version to use if not found
+ * @return {string} The PHP version to use in the composer.json file.
+ */
+function getComposerPhpVer(def) {
```

Comment:
Since we're not doing anything with the default version (`def`), would it be cleaner to just return `null` or `undefined` if we can't find any? And then change the callsite `getComposerPhpVer() ?? "8.3"`

Reply:
yer, null return is much cleaner, I have changed it